src/cmn/timerQueue.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020-present Infosys Limited  
+ *   
+ * SPDX-License-Identifier: Apache-2.0    
+ */
+
+#include "timerQueue.h"
+
+TimerContext::TimerContext(const CTime &expiryTime):
+    expiryTime_m(expiryTime)
+{
+
+}
+
+TimerContext::~TimerContext()
+{
+
+}
+
+void TimerContext::setExpiryTime(const CTime& expiryTime)
+{
+    expiryTime_m = expiryTime;
+}
+
+const CTime& TimerContext::getExpiryTime() const
+{
+    return expiryTime_m;
+}
+
+TimerQueue::TimerQueue():mutex_m(), container_m()
+{
+
+}
+
+TimerQueue::~TimerQueue()
+{
+
+}
+
+void TimerQueue::add(TimerContext* item)
```

Comment:
add/remove words are not good..should not it be addTimerInQueue, removeTimerInQueue,..Meaningful names make it easy to read.

Reply:
Updated function names.